metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -573,9 +573,17 @@ object MetalsEnrichments
         .filter(p => Files.exists(p.toNIO))
     }
     def targetroot: AbsolutePath = {
```

Comment:
We could do it the same as the method below `isSemanticdbEnabled`
```
def targetroot(scalaVersion: String):
```

and check one or the other flag based on the Scala version.

Reply:
I pushed the revised version. As you can see I had to pass the `scalaVersion` from call site. Tell me what you think.